### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by abror9109**

### CODE REVIEW

File: app/MoonShine/Resources/UserResource.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\MoonShine\Resources;
+
+use App\Models\User;
+// use Faker\Core\Number;
```
Comment (AnvarovD) [inline app/MoonShine/Resources/UserResource.php]: comment lani op tashen keremas bo'sa

File: app/Providers/AuthServiceProvider.php [hunk 2]
```diff
@@ -5,7 +5,8 @@
 // use Illuminate\Support\Facades\Gate;
 use Illuminate\Foundation\Support\Providers\AuthServiceProvider as ServiceProvider;
 
-class AuthServiceProvider extends ServiceProvider {
+class AuthServiceProvider extends ServiceProvider
+{
```
Comment (AnvarovD) [inline app/Providers/AuthServiceProvider.php:8]: CS-FIXER zapusk qilin

File: database/migrations/2023_07_19_044917_users_table_add_active_column.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->boolean('active')->nullable()->default(true)->after('is_admin');
```
Comment (AnvarovD) [inline database/migrations/2023_07_19_044917_users_table_add_active_column.php]: migratsiyala comment qo'shib ketin iloji boricha

            $table->boolean('active')->nullable()->default(true)->after('is_admin');

            $table->boolean('active')->nullable()->default(true)->after('is_admin')->comment("user Admin ekanligini bildirish uchun");

File: database/seeders/DatabaseSeeder.php [hunk 4]
```diff
@@ -20,5 +20,9 @@ public function run()
         //     'name' => 'Test User',
         //     'email' => 'test@example.com',
```
Comment (AnvarovD) [inline database/seeders/DatabaseSeeder.php]: comment lani o'chirvorin

**Review by parmonov98 — APPROVED** [review]
thanks!

**Pull request merged**